cmd/autoheal/metrics_exporter.go

```diff
@@ -39,3 +46,13 @@ func (h *Healer) incrementAwxActions(
 		},
 	).Inc()
 }
+
+func (h *Healer) incrementRequestedActions(type, rule, alert String) {
```

Comment:
I this valid syntax? I think that `type` can't be used as a parameter as it is a reserved word. Also `String` isn't a Go type. Maybe you pushed before committing your latest changes?

Reply:
@jhernand if *all* the parameters are of the same type than this is valid.